vendor/github.com/minio/minio-go/pkg/policy/bucket-policy-condition_test.go

```diff
@@ -0,0 +1,289 @@
+/*
```

Comment:
Can you remove this are you using 'govendor' tool here? it should have ignored the tests. We don't need this in vendorized directory.